libfuse/tlf.go

```diff
@@ -243,6 +243,15 @@ func (tlf *TLF) Setattr(ctx context.Context, req *fuse.SetattrRequest, resp *fus
 	return dir.Setattr(ctx, req, resp)
 }
 
+// Fsync implements the fs.NodeFsyncer interface for TLF.
+func (tlf *TLF) Fsync(ctx context.Context, req *fuse.FsyncRequest) (err error) {
+	dir, err := tlf.loadDir(ctx)
```

Comment:
If we don't have a `dir` yet, we wouldn't need to do a sync right? If so, perhaps we can just `getStoredDir()` like `Attr()`?